data_validation/config_manager.py

```diff
@@ -150,6 +150,17 @@ def append_query_groups(self, grouped_column_configs):
             self.query_groups + grouped_column_configs
         )
 
+    @property
+    def custom_query_type(self):
+        """Return SQL Query File from Config"""
+        return self._config.get(consts.CONFIG_CUSTOM_QUERY_TYPE, [])
+
+    def append_custom_query_type(self, custom_query_type):
```

Comment:
We shouldn't need to append the type to a list since custom_query_type should be a string value similar to [validation_type](https://github.com/GoogleCloudPlatform/professional-services-data-validator/blob/develop/data_validation/config_manager.py#L88)

Reply:
Done